content/blog/dotnet-preview/index.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Infrastructure as Code with .NET and Pulumi"
+authors: ["sophia-parafina"]
+tags: [". NET", "C#", "F#", "Visual Basic"]
+meta_image: "microsoft_logo.jpg"
```

Comment:
This looks like a very old .NET logo... I'm not sure where to find the official materials, but they look somewhat like this one there days: https://docs.microsoft.com/en-us/dotnet/images/hub/dotnet-core.svg

Reply:
replaced with current logo